java/maven/src/org/netbeans/modules/maven/execute/MavenCommandLineExecutor.java

```diff
@@ -523,7 +523,7 @@ private static List<String> createMavenExecutionCommand(RunConfig config, Constr
         if (config.getReactorStyle() != RunConfig.ReactorStyle.NONE) {
             File basedir = config.getExecutionDirectory();
             MavenProject mp = NbMavenProject.getPartialProject(config.getMavenProject());
-            File projdir = NbMavenProject.isErrorPlaceholder(mp) ? basedir : mp.getBasedir();
+            File projdir = mp == null || NbMavenProject.isErrorPlaceholder(mp) ? basedir : mp.getBasedir();
```

Comment:
Is this being cautious or did you see null here?  This would have thrown NPE in the previous release as far as I can tell.